### ISSUE TITLE
build.gradle should be x86_64 instead of x86-64

### ISSUE THREAD

**Issue by yikkok-yong** [facebook/react-native#23893]
## 🐛 Bug Report
A typo in build.gradle under android/app/ folder causing x86_64 apk not being build.

## To Reproduce
```
abi {
    ...
    include "armeabi-v7a", "x86", "arm64-v8a", "x86-64"
}
```

## Expected Behavior
```
abi {
    ...
    include "armeabi-v7a", "x86", "arm64-v8a", "x86_64"
}
```

## Code Example
N/A

## Environment
 React Native Environment Info:
    System:
      OS: Linux 4.15 Ubuntu 18.04.2 LTS (Bionic Beaver)
      CPU: (4) x64 Intel(R) Core(TM) i7-7500U CPU @ 2.70GHz
      Memory: 135.56 MB / 7.69 GB
      Shell: 4.4.19 - /bin/bash
    Binaries:
      Node: 10.15.2 - ~/.nvm/versions/node/v10.15.2/bin/node
      Yarn: 1.13.0 - /usr/bin/yarn
      npm: 6.8.0 - ~/.nvm/versions/node/v10.15.2/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      Android SDK:
        API Levels: 23, 26, 27, 28
        Build Tools: 23.0.1, 23.0.3, 26.0.3, 27.0.3, 28.0.1, 28.0.3
        System Images: android-23 | Intel x86 Atom_64, android-23 | Google APIs Intel x86 Atom_64
    npmPackages:
      react: 16.8.3 => 16.8.3 
      react-native: 0.59.0 => 0.59.0 
    npmGlobalPackages:
      react-native-cli: 2.0.1

**Comment by radeno** [comment]
@yikkok-yong you are right
cc @dulmandakh https://developer.android.com/ndk/guides/abis.html#sa

**Comment by grabbou** [comment]
Fix has already been released under 0.59.1.
